index.php

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Form</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <form id="form" action="index.php" method="post">
+        <label for="fname">First Name:</label> <br>
+        <input type="text" name="fname" id="fname" pattern="[A-za-z]{2-20}" required> <br>
+        <label for="lname">Last Name:</label> <br>
+        <input type="text" name="lname" id="lname" pattern="[A-za-z]{2-20}" required> <br>
+        
+        <label for="fullname">Full Name:</label> <br>
+        <input type="text" name="fullname" id="fullname" disabled required> <br>
```

Comment:
Why is it required field?

Reply:
To display full name, as I changed this file to html so I can not write php in it.